src/com/maddyhome/idea/vim/gn/VisualSelectNextSearch.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * IdeaVim - Vim emulator for IDEs based on the IntelliJ platform
+ * Copyright (C) 2003-2017 The IdeaVim authors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.maddyhome.idea.vim.gn;
+
+import com.intellij.openapi.actionSystem.DataContext;
+import com.intellij.openapi.editor.Editor;
+import com.maddyhome.idea.vim.VimPlugin;
+import com.maddyhome.idea.vim.action.motion.MotionEditorAction;
+import com.maddyhome.idea.vim.command.Argument;
+import com.maddyhome.idea.vim.handler.MotionEditorActionHandler;
+import org.jetbrains.annotations.NotNull;
+
+public class VisualSelectNextSearch extends MotionEditorAction {
+    public VisualSelectNextSearch() {
+        super(new Handler());
+    }
+
+    private static class Handler extends MotionEditorActionHandler {
+        public int getOffset(@NotNull Editor editor, DataContext context, int count, int rawCount, Argument argument) {
```

Comment:
Did you forget `@NotNull` before `DataContext context`?

Reply:
Yes, added.